email_templates/user_assigned_to_task.txt

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}{% trans "Task Assignment Alert from Metecho" %}
+
+{% blocktrans %}You have been assigned as the {{ role }} for "{{ task_name }}" on {{ repo_name }} project "{{ project_name }}". To view this task, please log into Metecho as user "{{ user_name }}". We appreciate your contribution to the Salesforce community. Thanks!{% endblocktrans %}
+
```

Comment:
Should we and is there any way to include the person who made the assignment? So, 

You have been assigned as the tester for Data Variable Stacks on Project Saturn by Michelle Lastname... 

Also, is this plain plain text or Rich text email? Wondering if we could make the task name and project name either bold or italic instead of using quotes?

Reply:
Hm, I'm pretty sure it's plain text. I think we could get the user's name who made the change though.